decidim-assemblies/app/views/decidim/assemblies/assemblies/_promoted_assembly.html.erb

```diff
@@ -5,7 +5,7 @@
         <%= link_to assembly_path(promoted_assembly), class: "card__link" do %>
           <h2 class="card__title"><%= translated_attribute promoted_assembly.title %></h2>
         <% end %>
-        <%== html_truncate(translated_attribute(promoted_assembly.short_description), length: 630, separator: '...') %>
+        <%= sanitize html_truncate(translated_attribute(promoted_assembly.short_description), length: 630, separator: '...') %>
```

Comment:
Not HTML input, no reason for sanitize.

Reply:
then we shouldn't use html_truncate here as well?

Reply:
just checked, the short description is actually HTML, so I'd leave that `sanitize` in.